packages/pipeline-editor/style/index.css

```diff
@@ -20,32 +20,15 @@
   text-align: center;
 }
 
-.save .list-item {
-  background-image: url('./save-pipeline.svg');
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 30%;
+.save .toolbar-icons:not(.overflow) {
+  height: 80%;
+  width: 80%;
 }
 
-.open .list-item {
-  background-image: url('./open-pipeline.svg');
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 43%;
-}
-
-.new .list-item {
-  background-image: url('./new-pipeline.svg');
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 30%;
-}
-
-.clear .list-item {
-  background-image: url('./clear-pipeline.svg');
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 40%;
+.new .toolbar-icons:not(.overflow),
```

Comment:
if you make the style for this selector the following: 
```
height: 90%;
width: 70%;
```
It appears in Safari like this: 
![image](https://user-images.githubusercontent.com/6673460/80733786-cbca8080-8ad3-11ea-8dd3-34f4df0e7345.png)
Chrome: 
![image](https://user-images.githubusercontent.com/6673460/80733931-03d1c380-8ad4-11ea-9aab-b386f53fe560.png)

Firefox:
![image](https://user-images.githubusercontent.com/6673460/80733879-eac91280-8ad3-11ea-99b0-b6e577e0ff05.png)

Reply:
Thanks, your idea made me realize that setting both width and height was the cause of the stretching. I need to override the the parent value for both, but I found that I could just set one to `auto` and then Chrome and FF will behave like Safari.